src/nullablevector.jl

```diff
@@ -1,3 +1,5 @@
+import Base: hcat, vcat
```

Comment:
Yeah, better not import anything and use the `Base.` prefix when overloading methods.